boot/freeldr/freeldr/arch/realmode/fathelp.inc

```diff
@@ -64,10 +64,20 @@ FatHelperEntryPoint:
     /* First save AX - the start cluster of freeldr.sys */
     push ax
 
+    /* Save ds */
+    push ds
+
+    /* Prepare ds before [bp-PutCharsOffset] call */
```

Comment:
```suggestion
    /* Prepare ds before PutCharsOffset call */
```
to simplify the comment (I think one can make the relation with the call done just below)